### PR TITLE
added links cleanup

### PR DESCRIPTION
**Pull request by tilast**

### CODE REVIEW

File: app/models/link.rb [hunk 1]
```diff
@@ -7,4 +7,12 @@ class Link < ActiveRecord::Base
   scope :without_issue, ->() { where(issue_id: nil) }
 
   delegate :title, to: :issue, prefix: true, allow_nil: true
+
+  before_save :cleanup_link
+
+  TEXT_TO_OMIT = ['?utm_source=rubyweekly&utm_medium=email']
```
Comment (tilast) [inline app/models/link.rb]: can move this to .env

Comment (rilian) [inline app/models/link.rb]: let's list only param names here

File: app/models/link.rb [hunk 2]
```diff
@@ -7,4 +9,19 @@ class Link < ActiveRecord::Base
   scope :without_issue, ->() { where(issue_id: nil) }
 
   delegate :title, to: :issue, prefix: true, allow_nil: true
+
+  before_save :cleanup_link
+
+  PARAMS_TO_OMIT = %w[utm_source utm_medium]
+
+  def cleanup_link
```
Comment (rilian) [inline app/models/link.rb:19]: is this a private method ?

File: app/models/link.rb [hunk 3]
```diff
@@ -7,4 +9,19 @@ class Link < ActiveRecord::Base
   scope :without_issue, ->() { where(issue_id: nil) }
 
   delegate :title, to: :issue, prefix: true, allow_nil: true
+
+  before_save :cleanup_link
+
+  PARAMS_TO_OMIT = %w[utm_source utm_medium]
```
Comment (rilian) [inline app/models/link.rb]: let's also reorganize file according to https://gist.github.com/rilian/8ca2ef7d52ca0181f0db

**Comment by tilast** [comment]
@rilian done

**Comment by rilian** [comment]
looks cool

**Pull request merged**